pkgs/by-name/wo/wordlists/package.nix

```diff
@@ -1,45 +1,54 @@
 { lib
-, callPackage
+, symlinkJoin
 , nmap
 , rockyou
-, runtimeShell
 , seclists
-, symlinkJoin
-, tree
 , wfuzz
 , lists ? [
     nmap
     rockyou
     seclists
     wfuzz
   ]
+, writeShellScriptBin
+, tree
 }:
+let
+  wordlistsCollection = symlinkJoin {
+    name = "wordlists-collection";
+    paths = lists;
 
-symlinkJoin rec {
-  pname = "wordlists";
-  version = "unstable-2023-10-10";
-
-  name = "${pname}-${version}";
-  paths = lists;
-
-  postBuild = ''
-    mkdir -p $out/bin
+    postBuild = ''
+      shopt -s extglob
+      rm -rf $out/!(share)
+      rm -rf $out/share/!(wordlists)
+      shopt -u extglob
+    '';
+  };
 
-    # Create a command to show the location of the links.
-    cat >> $out/bin/wordlists << __EOF__
-    #!${runtimeShell}
-    ${tree}/bin/tree ${placeholder "out"}/share/wordlists
-    __EOF__
-    chmod +x $out/bin/wordlists
+  wordlistsBinaries = let
```

Comment:
I thought a bit more about this and we could do:
```patch
From 07da9d4a824e0babd6ecf1607573088b565d5ad0 Mon Sep 17 00:00:00 2001
From: "Janik H." <janik@aq0.de>
Date: Fri, 17 Nov 2023 17:35:35 +0100
Subject: [PATCH] wordlists

---
 pkgs/by-name/wo/wordlists/package.nix | 28 +++++++++++----------------
 1 file changed, 11 insertions(+), 17 deletions(-)

diff --git a/pkgs/by-name/wo/wordlists/package.nix b/pkgs/by-name/wo/wordlists/package.nix
index a1f39e97eb26..5555077f13b5 100644
--- a/pkgs/by-name/wo/wordlists/package.nix
+++ b/pkgs/by-name/wo/wordlists/package.nix
@@ -26,28 +26,22 @@ let
     '';
   };
 
-  wordlistsBinaries = let
-    # A command to show the location of the links.
-    wordlistsBin = writeShellScriptBin "wordlists" ''
-      ${lib.getExe tree} ${wordlistsCollection}/share/wordlists
-    '';
-    # A command for easy access to the wordlists.
-    wordlistsPathBin = writeShellScriptBin "wordlists_path" ''
-      printf "${wordlistsCollection}/share/wordlists\n"
-    '';
-  in symlinkJoin {
-    name = "wordlists-binaries";
-    paths = [
-      wordlistsBin
-      wordlistsPathBin
-    ];
-  };
+  # A command to show the location of the links.
+  wordlistsBin = writeShellScriptBin "wordlists" ''
+    ${lib.getExe tree} ${wordlistsCollection}/share/wordlists
+  '';
+  # A command for easy access to the wordlists.
+  wordlistsPathBin = writeShellScriptBin "wordlists_path" ''
+    printf "${wordlistsCollection}/share/wordlists\n"
+  '';
+
 in symlinkJoin {
   name = "wordlists";
 
   paths = [
+    wordlistsBin
+    wordlistsPathBin
     wordlistsCollection
-    wordlistsBinaries
   ];
 
   meta = with lib; {
-- 
2.42.0
```

which feels a bit cleaner since we save on another symlinkJoin.